chunky/src/java/se/llbit/pfm/PfmFileWriter.java

```diff
@@ -75,19 +75,9 @@ private void writePixelData(Scene scene, ByteOrder byteOrder, TaskTracker.Task t
       ByteBuffer buffer = ByteBuffer.allocate(width*3*4).order(byteOrder);
       FloatBuffer floatBuffer = buffer.asFloatBuffer();
 
-      // get the row's data as floats
-      if (scene.postprocess == Postprocess.NONE)
-        // from raw pixel data
+      // get the row's data as floats from raw pixel data
```

Comment:
```suggestion
      // get the row's data as floats from raw pixel data
      // (ignore post processing because that would clip the color range and defeat the purpose of HDR)
```